manifest.json

```diff
@@ -49,6 +49,10 @@
             {
                 "name": "is_public",
                 "type": "boolean",
+		"help": {
+                        "en": "If enabled, Misskey will be accessible by people who doesn’t have an account. This can be changed later via the webadmin.",
+                        "fr": "Si cette case est cochée, Misskey sera accessible aux personnes n’ayant pas de compte. Vous pourrez changer ceci plus tard via la webadmin."
```

Comment:
```suggestion
                        "en": "If enabled, firefly-III will be accessible by people who doesn’t have an account. This can be changed later via the webadmin.",
                        "fr": "Si cette case est cochée, firefly-III sera accessible aux personnes n’ayant pas de compte. Vous pourrez changer ceci plus tard via la webadmin."
```